processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -996,6 +1091,22 @@ func (c *WatchClient) extractNamespaceLabelsAnnotations() bool {
 	return false
 }
 
+func (c *WatchClient) extractDeploymentLabelsAnnotations() bool {
```

Comment:
is this also to be checked with which metrics are enabled?

Reply:
not sure I understand...